test/extensions/filters/http/ext_authz/ext_authz_test.cc

```diff
@@ -154,6 +159,23 @@ TEST_F(HttpFilterTest, MergeConfig) {
   EXPECT_EQ("value", merged_extensions.at("key"));
 }
 
+class HttpExtAuthzFilterTestBase : public HttpFilterTest {
+public:
+  void initConfig(envoy::config::filter::http::ext_authz::v2::ExtAuthz& proto_config) {
+    config_ = std::make_unique<FilterConfig>(proto_config, local_info_, stats_store_, runtime_,
+                                             http_context_);
+  }
+};
+
+class HttpExtAuthzFilterTest : public HttpExtAuthzFilterTestBase {
```

Comment:
I'm not seeing any tests using these fixtures--what's their purpose?

Reply:
Good catch. They must have been needed in some earlier revision of this PR and never cleaned up after they became superfluous.  Gone.